### ISSUE TITLE
Map Explorer // VIIRS Fire data not displaying 

### ISSUE THREAD

**Issue by prustar** [OpenDevelopmentMekong/wp-odm_theme#469]
Perhaps this is potentially because there are too many data points? Or Map explorer can't manage to render the time series data. 

Test layers in both Carto and Geoserver neither load, filed under environment category in PP site.

![image](https://cloud.githubusercontent.com/assets/14996905/18833850/09b96258-83f3-11e6-9a68-b878dbd2e2fc.png)

Located in Carto @ https://odm.carto.com/viz/554ec2aa-7e74-11e6-be04-0ef7f98ade21/map
Located in Geoserver @ https://geoserver.opendevelopmentmekong.net/geoserver/web/?wicket:bookmarkablePage=:org.geoserver.web.data.resource.ResourceConfigurationPage&name=fire_nrt_V1_218&wsName=ODMekong

**Comment by acorbi** [comment]
@prustar some feedback about the geoserver layer: it seems that the layer is not correctly created on geoserver since it throws an error when you try to visualize it on the geoserver web UI.

Go to https://geoserver.opendevelopmentmekong.net/geoserver/web/?wicket:bookmarkablePage=:org.geoserver.web.demo.MapPreviewPage and click on GeoLayers, GeoTiff, PDF or whatever other format. I will throw following error:

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><!DOCTYPE ServiceExceptionReport SYSTEM "https://geoserver.opendevelopmentmekong.net/geoserver/schemas/wms/1.1.1/WMS_exception_1_1_1.dtd"> <ServiceExceptionReport version="1.1.1" >   <ServiceException>
      java.nio.BufferUnderflowException
</ServiceException></ServiceExceptionReport>
```
Can you try to fix the layer on geoserver?

The carto layer does not load indeed. Problem is that it is not throwing any error....

**Comment by prustar** [comment]
@acorbi When I tested the geoserver layer it spat out a wms file which I couldn't open whatsoever and didn't visualise. 

Lets put a pin in this one and I'll think of another way to represent the data.

**Comment by acorbi** [comment]
This should be fixed when https://github.com/OpenDevelopmentMekong/wp-odm_theme/issues/495 is fixed.

**Comment by acorbi** [comment]
Closing, fixed via #495
